en/components/banner.md

```diff
@@ -234,6 +234,101 @@ export class MyBannerComponent {
 > [!NOTE]
 > If the changes above are applied, the banner will never open, as the opening event is always cancelled.
 
+### Styling
```

Comment:
I think the styling section should h2 (`##` instead of `###`), so it appears on the right navigation right away, since it's a major section by itself:
![image](https://user-images.githubusercontent.com/24314672/61436136-d0f28080-a942-11e9-9a50-14a8253e39d6.png)